opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -61,9 +61,6 @@ RUN chmod +x root/startup/startup.d/*
 #
 FROM cafapi/opensuse-jre8:2
 
-RUN zypper -n refresh && \
-    zypper -n update
-
 ENV CATALINA_HOME /usr/share/tomcat
```

Comment:
After consulting with Michael on failing jenkins build this was removed due to some permissions issue.  I can revert and raise its own PR if required.

Reply:
We're going to need to do a release of this base image.  Create a separate ticket for that and include it is to remove this refresh and update but to restore it immediately after the release.